pandas/core/frame.py

```diff
@@ -1940,7 +1941,7 @@ def _sizeof_fmt(num, size_qualifier):
             mem_usage = self.memory_usage(index=True, deep=deep).sum()
             lines.append("memory usage: %s\n" %
                          _sizeof_fmt(mem_usage, size_qualifier))
-        _put_lines(buf, lines)
+        buffer_put_lines(buf, lines)
```

Comment:
same

Reply:
Good point, changed those. I'll take a look at the two `get_level_lengths`, but may not get around to fixing that today, so it's probably better as a new PR